sysbuild/Kconfig.matter

```diff
@@ -39,4 +39,7 @@ config MATTER_OTA_IMAGE_FILE_NAME
 	help
 	  Provides the file name of the generated Matter OTA image in the build directory.
 
+config SUIT_ENVELOPE_ROOT_TEMPLATE
+	default "${ZEPHYR_NRF_MODULE_DIR}/samples/matter/common/suit_templates/root_with_binary_nordic_top.yaml.jinja2"
```

Comment:
based on this I'm thinking your other paths are wrong

Reply:
Yeah, I think this should be the same in both places.

Reply:
As I wrote here: https://github.com/nrfconnect/sdk-nrf/pull/16214#discussion_r1666459774 there is a different approach to paths. I need to use different paths depending on the config location. @nordicjm maybe you could explain to me why I cannot use the same paths in `sysbuild/Kconfig.matter` and in `samples/matter/lock/boards/nrf54h20dk_nrf54h20_cpuapp.conf` ?

Reply:
I wanted to use the same paths here and there, but I couldn't :/

Reply:
@nordicjm Could you help me and explain why I need to use different paths?